src/HMRServer.js

```diff
@@ -47,8 +51,11 @@ class HMRServer {
       });
     }
 
-    const containsHtmlAsset = assets.some(asset => asset.type === 'html');
-    if (containsHtmlAsset) {
+    const isReload = this.options.reload;
+    const isReloadAsset = assets.some(asset => {
+      return isReload ? asset.type !== 'css' : asset.type === 'html';
```

Comment:
Actually I hope the modification for styles can also reload the page, because it is related to DOM.
So I am thinking would it be better to be like `isReload ? asset.type === 'js' : (asset.type === 'html' || asset.type === 'css')`?

Reply:
It sounds like you want reload to happen in any situation, so:

```js
const isReloadAsset = isReload ? true : assets.some(a => a.type === 'html');
```

During the PR I assumed it was desirable to have CSS apply without page reloads, since it doesn't depend on JavaScript state and the like. But because it's using `import`, maybe it runs into the same problems, and it's better to just hard reload the page?

Reply:
Oh yes. Maybe hot reload every time is more like what I need 😆